parts/linux/cloud-init/artifacts/containerd-monitor.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
@xuto2 we currently bake docker-monitor.service files into our VHD, hence I'm also adding a separate set of containerd-monitor units into our VHDs. 
What do you think about taking them out of our VHD builds and have them be dynamically templated via agentbaker and injected as cse data? We're currently doing this for raw ubuntu nodes only.